site/src/api/queries/settings.ts

```diff
@@ -0,0 +1,34 @@
+import * as API from "api/api";
+import {
+  type UserQuietHoursScheduleResponse,
+  type UpdateUserQuietHoursScheduleRequest,
+} from "api/typesGenerated";
+import { type QueryClient, type QueryOptions } from "@tanstack/react-query";
+
+export const userQuietHoursScheduleKey = (userId: string) => [
+  "settings",
+  userId,
+  "quietHours",
+];
+
+export const userQuietHoursSchedule = (
+  userId: string,
+): QueryOptions<UserQuietHoursScheduleResponse> => {
+  return {
+    queryKey: userQuietHoursScheduleKey(userId),
+    queryFn: () => API.getUserQuietHoursSchedule(userId),
+  };
+};
+
+export const updateUserQuietHoursSchedule = (
```

Comment:
I feel like the name might be a little confusing – I'd expect this to perform an update immediately, rather than generate options for a `useMutation` call

Reply:
this is what we've been going with so far. if it's an issue we can have a bigger discussion in slack. 🤷‍♀️